src/js/components/DonationPage/Donation.js

```diff
@@ -0,0 +1,165 @@
+<input type="hidden" name="hosted_button_id" value="WR3NGAATKPUWS" />
+<input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
+<img alt="" border="0" src="https://www.paypal.com/en_ZA/i/scr/pixel.gif" width="1" height="1" />
+</form> 
+*/
+class donation extends Component {
+  constructor(props){
+    super(props);
+    this.state ={
+      "cardNumber": "",
+      "nameonCard": "",
+      "expirationDate": "",
+      "cvc": "",
+      "country": "",
+      "state": "",
+      "city": "",
+      "postal": "",
+      "email": "",
+    }
+    
+   // this.onChange=this.onChange.bind(this);
+    //this.onSubmit=this.onSubmit.bind(this);
+  }
+
+  Payment(){
+
+    const RequestObject = {
+      "cardNumber": this.state.cardNumber,
+      "nameonCard": this.state.nameonCard,
+      "expirationDate": this.state.expirationDate,
+      "cvc": this.state.cvc,
+      "country": this.state.country,
+      "state": this.state.state,
+      "city": this.state.city,
+      "postal": this.state.emailValue,
+      "email": this.state.email,
+    }
+    fetch("https://localhost:44311/api/PayPal", {
+      method: 'post',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(RequestObject)
+    }).then(function(response) {
+      return response.json();
+    }).catch(function(data) {
+    });
+
+  }
+
+  render() {
+    return (
+      
+      <div className="App"> 
+                <div>
+                 
+                     <form>
+                     <h>Donation details</h><br/>
+                          Amount &nbsp;<input type="text" name="Amount" placeholder="R50.00"/><br/>
+
+                         
+                          <p><b>Organisation</b><br/> Please choose your organisation of choice</p>
+
+                        <br/>
+                          Organisation Name &nbsp;<input type="checkbox" name="SecondOrganisation" size="50%" />&nbsp;
+                          <navItem>
+                                <NavLink to="/register" ><button>view more</button></NavLink>
+                          </navItem><br/><br/>
+                          Organisation Name &nbsp;<input type="checkbox" name="SecondOrganisation" size="50%" />&nbsp;
+                          <navItem>
+                                <NavLink to="/register" ><button>view more</button></NavLink>
+                          </navItem>
+                     </form>
+                     
+                     
+                      <form>
+                      <h>Payments</h><br/>
+                     Credit card number&nbsp;<input type="number" name="card number" placeholder="1234567890"/><br/><br/>
+                     Expiration date&nbsp;<input type="date" name="ExpDate" placeholder="1234567890"/><br/><br/>
```

Comment:
The expiration field of credit cards is usually made of the month and the year in two separated fields.
e.g: exp month: MM  || exp year: YYYY